proto/xudrpc.proto

```diff
@@ -405,6 +413,30 @@ message ListPeersResponse {
   repeated Peer peers = 1 [json_name = "peers"];
 }
 
+message ListOrderHistoryRequest {
+  string order_id = 1 [json_name = "order_id"]; 
+}
+
+message ListOrderHistoryResponse {
+  repeated Trade trades = 1 [json_name = "trades"];
+  repeated SwapSuccess swaps = 2 [json_name = "swaps"];
+  OrderStatus status = 3 [json_name = "status"];
+
+  enum OrderStatus {
+    FILLED  = 0;
+    PARTIALLYFILLED = 1;
```

Comment:
enums should be `CAPITALS_WITH_UNDERSCORES` according to the [proto3 style guide](https://developers.google.com/protocol-buffers/docs/style).